src/puppet-web/puppet-web.ts

```diff
@@ -384,6 +389,8 @@ export class PuppetWeb extends Puppet {
     const first = cookie.find(c => c.name === 'webwx_data_ticket')
     const webwxDataTicket = first && first.value
     const size = buffer.length
+    const id = 'WU_FILE_' + this.fileId
```

Comment:
Here you save WU_FILE_fileId to id, but in the following code of `formData` you use `this.fileId` instead.

Is this a mistake, or by design?